tests/unit/karma.conf.js

```diff
@@ -60,9 +60,9 @@ module.exports = function(config) {
             'tests/unit/spec/base/search-params.js',
             'tests/unit/spec/base/mozilla-svg-image-fallback.js',
             'tests/unit/spec/base/mozilla-traffic-cop-funnelcake-geo.js',
-            'tests/unit/spec/firefox/features/sync.js',
             'tests/unit/spec/firefox/new-ios-redirect-helper.js',
             'tests/unit/spec/firefox/tracking-protection-tour.js',
+            'tests/unit/spec/firefox/whatsnew/whatsnew-61.js',
             'tests/unit/spec/ie8/mozilla-utils-ie8.js',
             'tests/unit/spec/base/send-to-device.js',
```

Comment:
The original `send-to-device.js` is pretty well unit tested, but I don't see any tests for the new replica lib added here. If we're going to maintain two versions of the JS or deprecate one over the other, it would be great not lose coverage?